app/src/main/java/mod/bobur/BoburUtils.java

```diff
@@ -0,0 +1,20 @@
+package mod.bobur;
```

Comment:
So unnecessary. Class `ThemeUtils` already have method to check current device theme.

Reply:
Yeah after 2-3 days I found ThemeUtils then I am not deleted my class